MigFiles/MIG/Interfaces/EmbeddedSystems/RaspiGPIO.cs

```diff
@@ -157,6 +157,11 @@ public string Domain
         }
 
         public List<MIGServiceConfiguration.Interface.Option> Options { get; set; }
+       
+        /// <summary>
+        /// get modules and module properties
+        /// </summary>
+        public List<InterfaceModule> Modules { get; set; }
```

Comment:
You're carrying old code in your repository. This also has been removed in current HG code.